_includes/pbc/all/install-features/202410.0/install-the-amazon-quicksight-module.md

```diff
@@ -0,0 +1,617 @@
+        'SprykerB2CMPDefaultDatasetMerchantOrder',
+        'SprykerB2CMPDefaultDatasetMerchantOrderCategory',
+        'SprykerB2CMPDefaultDatasetMerchantOrderItems',
+        'SprykerB2CMPDefaultDatasetMerchantProductOffer',
+        'SprykerB2CMPDefaultDatasetMerchantProductProductAbstract',
+        'SprykerB2CMPDefaultDatasetMerchantStore',
+        'SprykerB2CMPDefaultDatasetOrderDiscounts',
+        'SprykerB2CMPDefaultDatasetOrderItemCategoryProductBrand',
+        'SprykerB2CMPDefaultDatasetOrderItemLocalizedProductConcrete',
+        'SprykerB2CMPDefaultDatasetOrderItemProductCategory',
+        'SprykerB2CMPDefaultDatasetOrderItemsReturnDate',
+        'SprykerB2CMPDefaultDatasetOrderItemState',
+        'SprykerB2CMPDefaultDatasetOrderItemStateHistory',
+        'SprykerB2CMPDefaultDatasetOrderPaymentMethods',
+        'SprykerB2CMPDefaultDatasetOrderReturns',
+        'SprykerB2CMPDefaultDatasetOrderReturnsProductConcrete',
+        'SprykerB2CMPDefaultDatasetOrderShipmentMethods',
+        'SprykerB2CMPDefaultDatasetOrderTotalsCustomSQL',
+        'SprykerB2CMPDefaultDatasetProductConcreteAvailability',
+        'SprykerB2CMPDefaultDatasetProductConcreteStore',
+        'SprykerB2CMPDefaultDatasetQuoteProducts',
+    ];
+
+    /**
+     * @var string
+     */
+    protected const DEFAULT_DATA_SOURCE_ID = 'SprykerB2CMPDefaultDataSource';
+}
+```
+</details>
+
+<details>
+<summary>B2C</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\AmazonQuicksight;
+
+use SprykerEco\Zed\AmazonQuicksight\AmazonQuicksightConfig as SprykerEcoAmazonQuicksightConfig;
+
+class AmazonQuicksightConfig extends SprykerEcoAmazonQuicksightConfig
+{
+    /**
+     * @var list<string>
+     */
+    protected const ASSET_BUNDLE_IMPORT_DELETE_DATA_SET_IDS = [
+        'SprykerB2CDefaultDatasetCategoryLocalizedProductAbstract',
+        'SprykerB2CDefaultDatasetCustomer',
+        'SprykerB2CDefaultDatasetCustomerAddress',
+        'SprykerB2CDefaultDatasetOrderDiscounts',
+        'SprykerB2CDefaultDatasetOrderItemCategoryProductBrand',
+        'SprykerB2CDefaultDatasetOrderItemLocalizedProductConcrete',
+        'SprykerB2CDefaultDatasetOrderItemProductCategory',
+        'SprykerB2CDefaultDatasetOrderItemsReturnDate',
+        'SprykerB2CDefaultDatasetOrderItemState',
+        'SprykerB2CDefaultDatasetOrderItemStateHistory',
+        'SprykerB2CDefaultDatasetOrderPaymentMethods',
+        'SprykerB2CDefaultDatasetOrderReturns',
+        'SprykerB2CDefaultDatasetOrderReturnsProductConcrete',
+        'SprykerB2CDefaultDatasetOrderShipmentMethods',
+        'SprykerB2CDefaultDatasetOrderTotalsCustomSQL',
+        'SprykerB2CDefaultDatasetProductConcreteAvailability',
+        'SprykerB2CDefaultDatasetProductConcreteStore',
+        'SprykerB2CDefaultDatasetQuoteProducts',
+    ];
+
+    /**
+     * @var string
+     */
+    protected const DEFAULT_DATA_SOURCE_ID = 'SprykerB2CDefaultDataSource';
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
```

Comment:
Move this to the step where this can be verified.
To this box add: `These changes are verified in a later step`.